### DISCUSSION TITLE
SEB for iOS

### DISCUSSION THREAD

**Discussion by danschlet** [SafeExamBrowser/seb-mac#96] [General]
With the release of the major update SEB 3.0 for iOS, we would like to move all discussions and issues about SEB for iOS to GitHub. Please don't use our SourceForge discussions and tickets anymore.

**Comment by mkendrick** [comment]
This is excellent news.  Do you allow the app to be downloaded/run on M1 macs? 

Will there be a follow-up version for OSX that supports webkit/access to camera/mic etc?

**Reply by danschlet** [reply]
The last feedback from Apple was, that the iOS Assessment mode (AAC) isn't correctly running on M1 Macs, therefore they advise to not allow apps using AAC to run on M1 Macs. This might change with new macOS versions, but it anyways doesn't make much sense, as the native macOS 3.0 version will offer a better user experience.

**Comment by TestingUserFromTheField** [comment]
I was very happy to read that Safe Exam Browser 3.0 for iOS has been released and that it supports the modern WebKit browser engine, with access to camera/microphone via WebRTC on iOS 14.3. I'm testing a moodle assessment quiz where students have to register themselves (audio) on a iOS 14.6 iPad. The audio recording works well without SEB, but with SEB (just dowloaded from the store), it still says that it does NOT support WebRTC. 
Could you help me and tell what the problem is?

**Reply by danschlet** [reply]
Unfortunately there are some restrictions for when the modern WebView can be used. These are mentioned in the SEB 3.0 for iOS [release notes](https://safeexambrowser.org/ios/ios_release_notes_en.html). In short, as long as the Moodle SEB integration isn't updated yet, you can only use WebRTC in separate browser tabs which are not running on the same domain as the Moodle server. We are currently discussing with the Moodle guys when the Moodle SEB integration can be upgraded to support SEB 3.0 for iOS and macOS with the modern WebKit browser engine.

**Comment by TestingUserFromTheField** [comment]
Hi,

Thank you for your answer. I assume this means the problem will be solved by using moodle 3.9?
Best regard

**Reply by danschlet** [reply]
No, the modern SEB integration available from Moodle 3.9 (and as plugin for Moodle 3.7 and 3.8) needs to be updated.

**Comment by TestingUserFromTheField** [comment]
Oh, thank you! I'll See with the IT then..

**Comment by mkendrick** [comment]
Thanks Daniel.  Our use case dones not involve a LMS integration at all, but we do need to be able to launch Zoom as an allowed application (whether it’s the zoom client itself or the web-based client).
